spec/features/saml/loa3_sso_spec.rb

```diff
@@ -66,14 +75,38 @@ def sign_out_user
 
   context 'canceling verification' do
     context 'with js', js: true do
+      let(:warning_qualifier) { t('idv.cancel.warning_qualifier') }
+      let(:sp_name) { 'Your friendly Government Agency' }
+
+      it 'does not show the service provider name if not signed up via a service provider' do
+        sign_in_and_2fa_user
+        loa3_sp_session
+
+        visit verify_path
+        click_idv_begin
+        click_on t('links.cancel')
+
+        modal = page.find('.modal-warning')
+        expect(modal).to have_content(
+          t(
+            'idv.cancel.warning_point_no_sp',
+            warning_qualifier: warning_qualifier,
+            sp_name: sp_name
```

Comment:
This expectation contradicts the title of this test, which says that it should not show the service provider name.

Reply:
I see where I got confused. The `warning_point_no_sp` entry doesn't use the `sp_name` variable, so we don't need to include it in the expectation.